### PR TITLE
Qa: some code style reflected for pep8

### PR DESCRIPTION
**Pull request by RyanKung**
I just read the test code and reflected it to [https://www.python.org/dev/peps/pep-0008/](pep8) code style.
There is partial of them, and if this pull request be accepted, I will continue to reflect all of them done.

(I'm not a native English speaker, and sorry for my bad English)

### CODE REVIEW

File: qa/pull-tester/rpc-tests.py [hunk 1]
```diff
@@ -31,33 +31,33 @@
...
+
 # python-zmq may not be installed. Handle this gracefully and with some helpful info
 if ENABLE_ZMQ:
     try:
         import zmq
     except ImportError:
-        print("WARNING: \"import zmq\" failed. Setting ENABLE_ZMQ=0. " \
-            "To run zmq tests, see dependency info in /qa/README.md.")
-        ENABLE_ZMQ=0
+        print('''WARNING: \"import zmq\" failed. Setting ENABLE_ZMQ=0. "
```
Comment (MarcoFalke) [inline qa/pull-tester/rpc-tests.py]: I don't think they are equivalent.

**Comment by MarcoFalke** [comment]
I appreciate that the code is being cleaned up but some of the patches will conflict with my pull requests to refactor to python3, so I'd prefer to hold this back until the switch is done.

File: qa/pull-tester/rpc-tests.py (hunk 1 above)
Comment (RyanKung) [inline qa/pull-tester/rpc-tests.py]: @MarcoFalke yes, you are right, review & fixing

**Comment by RyanKung** [comment]
@MarcoFalke Ok

**Pull request closed without merging**